cpg-analysis/src/main/kotlin/de/fraunhofer/aisec/cpg/query/FlowQueries.kt

```diff
@@ -430,12 +432,12 @@ fun Node.alwaysFlowsTo(
                         value = true,
                         children = mutableListOf(QueryTree(value = it)),
                         stringRepresentation =
-                            "The EOG path reached the node ${it.lastOrNull()} matching the required predicate" +
+                            "The EOG path reached the node ${it.nodes.lastOrNull()} matching the required predicate" +
                                 if (earlyTermination != null)
                                     " before reaching a node matching the early termination predicate"
                                 else "",
                         node = this,
-                        terminationReason = Success(it.last()),
+                        terminationReason = Success(it.nodes.last()),
```

Comment:
Shouldn't we add the assumptions to the QueryTree? That should be quite easy

Reply:
Yes, I added it to the QueryTree in the `SinglePathResult` normally I would have expected the NodePath be a subgraph and therefore child of the QueryTree, but it is part of the value apparently